src/pages/ScoreReport.vue

```diff
@@ -598,6 +604,28 @@ const getScoresAndSupportFromAssessment = ({
   };
 };
 
+const computedProgressData = computed(() => {
+  if (!assignmentData.value) return [];
+  return assignmentData.value.map(({ assignment }) => {
+    const progress = assignment.assessments.reduce((acc, assessment) => {
+      const status = assessment.optional
+        ? 'optional'
+        : assessment.completedOn !== undefined
+        ? 'completed'
+        : assessment.startedOn !== undefined
+        ? 'started'
+        : 'assigned';
```

Comment:
We should be able to simplify this by removing the `!== undefined` checks, as the values themselves will be evaluated as truthy or falsy.

```suggestion
      const status = assessment.optional
        ? 'optional'
        : assessment.completedOn
        ? 'completed'
        : assessment.startedOn
        ? 'started'
        : 'assigned';
```